snowfakery/output_streams.py

```diff
@@ -35,6 +35,11 @@ def noop(x):
     return x
 
 
+def format_datetime(dt: datetime.datetime):
+    """Format into the Salesforce-preferred syntax."""
+    return str(dt).replace(" ", "T")
```

Comment:
What about `isoformat()` instead?
```suggestion
    return dt.isoformat(timespec="seconds")
```